src/cmds/whoami.js

```diff
@@ -1,24 +1,27 @@
 const chalk = require('chalk');
 const config = require('config');
 const configStore = require('../lib/configstore');
-const loginCmd = require('./login');
 
-exports.command = 'whoami';
-exports.usage = 'whoami';
-exports.description = 'Displays the current user and project authenticated with ReadMe.';
-exports.category = 'admin';
-exports.position = 3;
+module.exports = class WhoAmICommand {
+  constructor() {
+    this.command = 'whoami';
+    this.usage = 'whoami';
+    this.description = 'Displays the current user and project authenticated with ReadMe.';
+    this.category = 'admin';
+    this.position = 3;
 
-exports.args = [];
-
-exports.run = () => {
-  if (!configStore.has('email') || !configStore.has('project')) {
-    return Promise.reject(new Error(`Please login using \`${config.get('cli')} ${loginCmd.command}\`.`));
```

Comment:
I stopped directly calling the login command for its name here because we know it. It's login.